### PR TITLE
Issue 16778 change startup shutdown to include opendistro

### PR DESCRIPTION
**Pull request by dsilvam**

### CODE REVIEW

**Review by nollymar — APPROVED** [review]

File: bin/dotStartup.bat [hunk 1]
```diff
@@ -82,6 +82,59 @@ shift
...
+			echo skipping open distro
+			set LAUNCH_OPEN_DISTRO=false
+		)
+	)
+
+	echo Launching Open Distro? = !LAUNCH_OPEN_DISTRO!
+
+	if "!LAUNCH_OPEN_DISTRO!"=="true" (
+		rem Launching Open Distro...
+		docker run -d --name dot_opendistro -e PROVIDER_ELASTICSEARCH_HEAP_SIZE=1500m -e PROVIDER_ELASTICSEARCH_DNSNAMES=elasticsearch -e ES_ADMIN_PASSWORD=%OPEN_DISTRO_PASSWORD% -e discovery.type=single-node -p 9200:9200 gcr.io/cicd-246518/es-open-distro:1.2.0
```
Comment (wezell) [inline bin/dotStartup.bat:128]: if this fails, we should send a message saying something like "Unable to start the Elasticseach docker container, please make sure you either start Elasticseach before you start dotCMS or have docker available so dotCMS can start the Elasticseach docker container"

File: bin/dotStartup.bat [hunk 2]
```diff
@@ -82,6 +82,59 @@ shift
...
+set ELASTICSEARCH_PORT=9200
+set LAUNCH_OPEN_DISTRO=true
+set OPEN_DISTRO_ALREADY_RUNNING=false
+set OPEN_DISTRO_USER=admin
+set OPEN_DISTRO_PASSWORD=admin
+
+set health_check=stopped
+
+rem let's check if there's an Open Distro running
+for /f %%i in ('curl "%ELASTICSEARCH_HOST%:%ELASTICSEARCH_PORT%/_cat/health?h=status" -u %OPEN_DISTRO_USER%:%OPEN_DISTRO_PASSWORD% --insecure') do set health_check=%%i
```
Comment (wezell) [inline bin/dotStartup.bat]: does window have curl?

File: bin/dotStartup.sh [hunk 3]
```diff
@@ -124,6 +124,61 @@ your dotCMS application."
...
+                ;;
+        esac
+        shift
+    done
+
+    echo "Launching Open Distro? = $LAUNCH_OPEN_DISTRO"
+
+    if [ "$LAUNCH_OPEN_DISTRO" = true ] ; then
+        ## Bring up Open Distro
+        docker run -d --name dot_opendistro -e PROVIDER_ELASTICSEARCH_HEAP_SIZE=1500m -e PROVIDER_ELASTICSEARCH_DNSNAMES=elasticsearch -e ES_ADMIN_PASSWORD=$OPEN_DISTRO_PASSWORD -e discovery.type=single-node -p $OPEN_DISTRO_PORT:9200 gcr.io/cicd-246518/es-open-distro:1.2.0
```
Comment (wezell) [inline bin/dotStartup.sh:159]: if this fails, we should send a message saying something like "Unable to start the Elasticseach docker container, please make sure you either start Elasticseach before you start dotCMS or have docker available so dotCMS can start the Elasticseach docker container"

**Review by freddyucv — APPROVED** [review]

File: bin/dotStartup.bat (hunk 2 above)
Comment (dsilvam) [reply bin/dotStartup.bat]: yes

File: bin/dotStartup.bat (hunk 1 above)
Comment (dsilvam) [reply bin/dotStartup.bat:128]: Done

File: bin/dotStartup.sh (hunk 3 above)
Comment (dsilvam) [reply bin/dotStartup.sh:159]: Done

**Pull request merged**